### DISCUSSION TITLE
Collection of requirements, ideas, and features for Harbor 2.10

### DISCUSSION THREAD

**Discussion by Vad1mo** [goharbor/harbor#19238] [Ideas]
We invite the community to comment and vote on the next feature for Harbor 2.10

Here are some previous requirements to choose from:

- [Candidate 2.10.0  Issues](https://github.com/goharbor/harbor/issues?q=is%3Aissue+is%3Aopen+label%3Acandidate%2F2.10.0)
- https://github.com/goharbor/harbor/discussions/18396
- [Requirements flagged issues](https://github.com/goharbor/harbor/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Frequirement)

Please also use the up arrow and the thumbs  ↑ 👎 to help us rank your requests.

**Comment by IwanSE** [comment]
Hello! Is it possible to redesign the main repositories page to add favorites tabs? At the moment I have **270+ repositories** and I would like to have a **"favorites"** button, this will help better and easier to navigate.

I would be grateful if this is already being worked out, any feedback is welcome!

Cheers! @Vad1mo @OrlinVasilev 😇🍻

**Reply by AllForNothing** [reply]
Can you open a new issue as a feature request?

**Reply by IwanSE** [reply]
Hi! Sorry for the long wait! - #19517

**Comment by nbjohnson** [comment]
It would be great to be able to get priority on a security bug fix: https://github.com/goharbor/harbor/issues/15283 This has been around for a while and prevents us from being able to enable blocking enforcement for images

**Comment by felixtran39** [comment]
Hello! It would be great if we could address this issue: #18944 . This has been a pain-point for us for a while now.

**Comment by tostt** [comment]
Hello!

Replicating artifacts across two Harbor instances currently requires a system admin role, but this is typically the role of a project maintainer.

Most of the time there is no reason to involve the system admin in day-to-day project operations. The main reasons are :

- The project maintainer knows better which artifacts are needed where : shortening the loop makes it more efficient, safer and faster
- The system admin’s role is to maintain Harbor so that projects admins and maintainers can do their job, not to do it for them. 
- Giving permission to replicate artifacts should be possible without giving full control over the instance

The current situation doesn’t scale well as the number of projects increases.

This is proposed in issues https://github.com/goharbor/harbor/issues/18447 and https://github.com/goharbor/harbor/issues/13543.

**Comment by gaglimax** [comment]
Hello ! 

It would be great to trigger an event based replication when a system label is added to an artifact. This is mention is issue #13015.
